src/universal/config/myarea-datasets.ts

```diff
@@ -64,6 +64,24 @@ if (month !== 11 && month !== 0) {
 }
 
 export const DATASETS: DatasetCategories = {
+  laadpalen: {
+    title: 'Laadpalen',
+    datasets: {
+      laadpalen: {
+        title: 'Laadpalen',
+        filters: {
+          connector_type: {
```

Comment:
De values in connector_type lijken `;` separated te zijn. Kunnen we dit, denk je, op een of andere manier inzetten om wat mooiere filtering te maken? Je zou in de transformList functie deze values kunnen opslitsen in kunnen toevoegen aan de properties van een Feature:

type1: true/false
type2: true/false

etc.

Mogelijk nog een "Leesbare" naam bij die type nummers zetten. 

IEC_62196_T2 =  Bijvoorbeeld een zgn "Mennekes" stekker.
zie ook: https://nl.wikipedia.org/wiki/IEC_62196_Type_2

Reply:
ja het is te lelijk nu hmmm